kivy/uix/accordion.py

```diff
@@ -1,4 +1,4 @@
-'''
+﻿'''
```

Comment:
also, check your editor's config, there is an invisible character (U+FEFF) that was introduced at the beginning of the file here, had to use hexedit to replace it by something visible and edit it out later.